tests/models/test_dag.py

```diff
@@ -203,6 +203,12 @@ def test_dag_topological_sort(self):
 
         self.assertEqual(tuple(), dag.topological_sort())
 
+    def test_dag_naive_start_date_string(self):
+        DAG('DAG', default_args={'start_date': '2019-06-01'})
+
+    def test_dag_naive_start_end_dates_strings(self):
+        DAG('DAG', default_args={'start_date': '2019-06-01', 'end_date': '2019-06-05'})
```

Comment:
Could you add a test case that covers a non-default TZ start_date and a string end_date to check that the TZ is applied correctly to end_date?

Reply:
Great suggestion @ashb.  Test added.